packages/smarthr-ui/src/components/SideNav/stories/SideNav.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import { action } from '@storybook/addon-actions'
+import React, { ComponentProps, useState } from 'react'
+
+import { StatusLabel } from '../../StatusLabel'
+import { SideNav } from '../SideNav'
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+type items = ComponentProps<typeof SideNav>['items']
+
+export const SideNavItems: items = [
+  {
+    id: 'id-1',
+    title: ' one!',
+    isSelected: true,
+  },
+  {
+    id: 'id-2',
+    title: 'two!',
+    isSelected: false,
+  },
+  {
+    id: 'id-3',
+    title: 'three!',
+    isSelected: false,
+  },
+  {
+    id: 'id-4',
+    title: 'four!',
+    isSelected: false,
+    prefix: <StatusLabel>default</StatusLabel>,
+  },
+  {
+    id: 'id-5',
+    title: 'five!',
+    isSelected: false,
+    prefix: <StatusLabel type="blue">blue</StatusLabel>,
+  },
+]
+
+export default {
+  title: 'Navigation（ナビゲーション）/SideNav',
+  component: SideNav,
+  render: (args) => (
+    <div style={{ width: '200px' }}>
+      <SideNav {...args} />
+    </div>
+  ),
+  args: {
+    size: 'default',
+    items: SideNavItems,
+  },
+  excludeStories: ['SideNavItems'],
+  parameters: {
+    chromatic: { disableSnapshot: true },
+  },
+} satisfies Meta<typeof SideNav>
+
+export const Playground: StoryObj<typeof SideNav> = {
+  args: {},
+}
+
+export const Size: StoryObj<typeof SideNav> = {
```

Comment:
props は小文字にしたいです。他コンポーネントに合わせてもらえると嬉しいです。

```suggestion
export const Size: StoryObj<typeof SideNav> = {
  name: 'size',
```